portfolio/pom.xml

```diff
@@ -8,6 +8,7 @@
   <version>1</version>
   <packaging>war</packaging>
 
+
```

Comment:
Perhaps you can revert this file. There doesn't seem to be anything changing.

Reply:
Got it. Added something to make re-running the project easier and it didn't work so I deleted it, must have left a few spaces or something.